app/views/wiki/show.html.erb

```diff
@@ -105,6 +105,10 @@
     <div class="tab-pane active" id="tab-overview">
       <div<% unless @node.has_tag('style:wide') %> style="overflow:hidden;max-width:800px;"<% end %> id="content">
       <%= raw auto_link(insert_extras(@node.latest.render_body), :sanitize => false) %>
+      <div<% unless params[:raw] %> style="display:none;"<% end %> id="content-raw-markdown">
+        <%= raw auto_link(insert_extras(@node.latest.render_body), :sanitize => false) %>
+      </div>
+      </div>
       </div>
     </div>
```

Comment:
Ah, see how this is still showing the html, not the raw markdown? If you look in /app/models/drupal_node_revision.rb, you'll see you can use @node.latest.body instead and get the raw markdown. auto_link will also not be needed, not insert extras. Make sense?